File: Telusko/LinearSearch.java
import java.util.Scanner;

public class LinearSearch{

    public static int Linear(int arr[], int k){
        for(int i=0; i<arr.length; i++){
            if(arr[i] == k) return i;
        }
        return -1;
    }
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int arr[] = new int[n];
        for(int i=0; i<n; i++){
            arr[i] = sc.nextInt();
        }
        int k = sc.nextInt();
        sc.close();
        System.out.println(Linear(arr, k)+1);
    }
}

File: Apna/Day8/LinearSearch.cpp
#include<iostream>
using namespace std;

void LinearSearch(int n, int arr[], int k){
    for(int i=0; i<n; i++){
        if(arr[i] == k){
            cout<<i+1;
            return;
        }
    }
    cout<<"-1";
    return;
}
int main(){
    int n;
    cin>>n;
    int arr[n];
    for(int i=0; i<n; i++){
        cin>>arr[i];
    }
    int k;
    cin>>k;
    LinearSearch(n,arr,k);
}